draft-ietf-quic-transport.md

```diff
@@ -1415,31 +1415,59 @@ TODO: see issue #161
 Connections should remain open until they become idle for a pre-negotiated
 period of time.  A QUIC connection, once established, can be terminated in one
 of four ways: negotiated shutdown, idle timeout, immediate close, and a
```

Comment:
This list still refers to "negotiated shutdown" while the subsections refer to "application close."  Might want to align them.